pkg/actors/actors.go

```diff
@@ -0,0 +1,137 @@
+package actors
```

Comment:
Do we need this package? We can use the same Instances package and create Get(id string, port int) method in that package.

Reply:
Use the same instance package and create just various required types 

ActorsOutput, MetadataActiveActorsCount.

You can do the similar change for status.go as well.